public/theme/extensions-2021/partials/_footer.twig

```diff
@@ -0,0 +1,30 @@
+<footer>
+    <nav class="footer-nav">
+        <p>
+            &copy; {{ "now"|date("Y") }} &bullet;
+            {{ __('general.phrase.built-with-bolt', {'%url%':'https://bolt.cm'}) }}
+        </p>
+{#        {% for item in menu_array('footer') %}#}
```

Comment:
Comments! 🙈

Reply:
Yes, the footer is empty. Maybe we can fill it like the Bolt website or leave it like it is.

Reply:
ah, good point! Ok, let's leave it as-is for now, we can always uncomment it later if we add a menu! 👍